package.json

```diff
@@ -53,7 +53,10 @@
     "test:production": "cypress open --env testEnv=production",
     "test:master": "cypress open --env testEnv=master ",
     "test:branch": "cypress open --env testEnv=branch",
-    "test:local": "cypress open --env testEnv=local"
+    "test:local": "cypress open --env testEnv=local",
+    "test:local:dataflow": "cypress open --env testEnv=local:dataflow",
+    "test:cypress:clue": "cypress run --spec 'cypress/integration/clue/**/* --config video=false,defaultCommandTimeout=8000 --env testEnv=local",
+    "test:cypress:dataflow": "cypress run --spec 'cypress/integration/dataflow/**/*' --config video=false,defaultCommandTimeout=8000 --env testEnv=local"
```

Comment:
Consider adding
```
    "test:cypress:ci": "npm run test:cypress:dataflow"
```
and then `travis.yml` can just `npm run "test:cypress:ci"` so app-specific project configuration is limited to `package.json`.